src/proto/message_translation/tbots_protobuf.h

```diff
@@ -126,6 +126,29 @@ std::unique_ptr<TbotsProto::NamedValue> createNamedValue(const std::string name,
 std::unique_ptr<TbotsProto::PlotJugglerValue> createPlotJugglerValue(
     const std::map<std::string, double>& values);
 
+/**
+ * Returns a TbotsProto::DebugShapesMap proto containing the name
+ * shape pairs.
+ *
+ * Could use LOG(VISUALIZE) to plot these values. Example:
+ *  LOG(VISUALIZE) << *createDebugShapesMap({
+ *      {"circle_name", *createShapeProto(circle_object)},
+ *      {"stadium_name", *createShapeProto(stadium_object)},
+ *      {"polygon_name", *createShapeProto(polygon_object)}
+ *  });
+ *
+ * @param named_shapes The map of name shape proto pairs to plot
+ *
+ * @return The unique_ptr to a TbotsProto::DebugShapesMap proto containing data with
+ *        specified names and shapes
+ */
+std::unique_ptr<TbotsProto::DebugShapesMap> createDebugShapesMap(
+    const std::map<std::string, TbotsProto::Shape>& named_shapes);
+
+std::unique_ptr<TbotsProto::Shape> createShapeProto(const Circle& circle);
+std::unique_ptr<TbotsProto::Shape> createShapeProto(const Polygon& circle);
+std::unique_ptr<TbotsProto::Shape> createShapeProto(const Stadium& circle);
```

Comment:
nit: all the parameters all called circle

Reply:
Updated!